ruka/rook-ceph/rook-ceph-node-cleanup.sh

```diff
@@ -6,7 +6,7 @@ SSH_USER=jhoblitt_b
 
 for n in $(seq 1 3);
```

Comment:
This is special use script but it may make sense to adjust the range to 2-4 here just in case someone forgets and tries to run it as-is.

Reply:
range updated